src/routes/contact.js

```diff
@@ -191,12 +191,71 @@ router.post('/send', async (req, res) => {
       email.sendContactCheckInEmail(e)
     })
 
-    response.setMessage('Contacts emailed')
+    response.setMessage({
+      results: {
+        message: 'Contacts emailed',
+        total: emails.length
+      }
+    })
```

Comment:
The e-mail transmission appears to be asynchronous, without waiting for the results. If that's correct, all we can do here is indicate that's how many e-mail messages we attempted to send. 

If so, do we need to clarify that in the message?

Reply:
I think it would be confusing for the user if we change the message to something like "We attempted to email XXX contacts".  In my experience, people don't need literal explanations for these sorts of messages.  My opinion is that less is more, and we should just give a short and easy-to-consume message.